source/blog/2016-11-02-ovirt-provider-ovn.md

```diff
@@ -51,6 +51,8 @@ After installing the driver and OVN, the OVN-controller must be configured. This
     vdsm-tool ovn-config <OVN central server IP> <local IP used for OVN tunneling>
 
 The second parameter (local IP used for OVN tunneling) can be the IP address of the ovirtmgmt interface on the host, and should be reachable by OVN hosts and central server.
+vdsm-tool will also configure ovn-controller with the PKI required to safely connect to the ovn central server.
+To add support for Transport Layer Security (TLS) connectivity with the OVN-controller, vdsm-tool will use the VDSM private and public keys, and configures OVN-controller to enable PKI encryption between the host and the OVN south db on the OVN central server. This provides privacy and data-integrity in the management connections to the south database
```

Comment:
I'm not really sure about this one.
Is OVN-controller a thing, or is this a name of a thing?
Note that for we do not use "the" for vdsm and ovirt-engine, as these are names. I think the same rule applies here, so I would rather not have it here either.